contracts/governance/StakingRewards/StakingRewards.sol

```diff
@@ -31,7 +30,7 @@ contract StakingRewards is StakingRewardsStorage, Initializable {
 	 * @param _SOV SOV token address
 	 * @param _staking StakingProxy address should be passed
 	 * */
-	function initialize(address _SOV, IStaking _staking) external onlyOwner initializer {
+	function initialize(address _SOV, IStaking _staking) external onlyOwner {
```

Comment:
They are just placeholders. The issue was with inheritance order which would otherwise have problems with storage. The initialize funtion is already triggered while inital deployment and hence won't be necessary again.